source/extensions/tracers/opentelemetry/tracer_stats.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "envoy/stats/stats_macros.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace OpenTelemetry {
+
+#define OPENTELEMETRY_TRACER_STATS(COUNTER)                                                        \
+  COUNTER(http_reports_failed)                                                                     \
+  COUNTER(http_reports_sent)                                                                       \
+  COUNTER(http_reports_success)                                                                    \
```

Comment:
Hmmm... if an Envoy upstream cluster is used, it should have these metrics, right?
When are these expected to be used, and why would it be different than gRPC-Service?

Reply:
To be honest, this type was just brought-over from the original PR from @AlexanderEllis. I'm not sure what he had in mind, but I guess this is to offer some stats on the number of exports OTLP requests. You are right, probably shouldn't be different if one is using OTLP/gRPC vs OTLP/HTTP. 

I'm not familiar how this works, and given the PR is already big, do you think we need this right now? Would this be something that can be added as a follow up?

Reply:
I suggest keeping the old counters (keeping consistency with prior versions), and avoid introducing the new ones for now. If possible, I'd like the integration tests to take do look at counters that are inherent with the upstream clusters.

Reply:
I removed the new type and kept the existing counters as is. [ba5d943](https://github.com/envoyproxy/envoy/pull/29207/commits/ba5d94396faa2fd2ddf8d0d1064f97790706f2c6)